fmriprep/cli/parser.py

```diff
@@ -40,10 +40,19 @@ def _drop_sub(value):
         value = str(value)
         return value.lstrip('sub-')
 
+    def _filter_pybids_none_any(dct):
+        import bids
+        for k, v in dct.items():
+            if v is None:
+                dct[k] = bids.layout.Query.NONE
```

Comment:
PyBIDS should already treat `None` like `Query.None`.

Reply:
Weird, I tried and it did not, and now it does. 
Maybe my sleep deprived brain is just making a lot of typos. :/
I will remove it.